bigbluebutton-html5/private/config/settings.yml

```diff
@@ -759,6 +759,7 @@ public:
   whiteboard:
     annotationsQueueProcessInterval: 60
     cursorInterval: 150
+    pointerDiameter: 10
```

Comment:
If this was kept as `5`, I think the pull request would just be making the size configurable.

Given that you're changing the size, we get to talk about should we mirror the change on playback or video format...

Reply:
Done! Thanks for the suggestion.